ESSArch_Core/tags/serializers.py

```diff
@@ -641,14 +644,18 @@ class Meta:
 
 class DeliverySerializer(serializers.ModelSerializer):
     type = DeliveryTypeSerializer()
+    submission_agreement = SubmissionAgreementSerializer()
+    producer_organization = AgentSerializer()
 
     class Meta:
         model = Delivery
-        fields = ('id', 'name', 'type', 'description')
+        fields = ('id', 'name', 'type', 'description', 'submission_agreement', 'producer_organization')
 
 
 class DeliveryWriteSerializer(DeliverySerializer):
     type = serializers.PrimaryKeyRelatedField(queryset=DeliveryType.objects.all())
+    submission_agreement = serializers.PrimaryKeyRelatedField(queryset=SubmissionAgreement.objects.all())
```

Comment:
We should only allow published submission agreements here, right?

```suggestion
    submission_agreement = serializers.PrimaryKeyRelatedField(queryset=SubmissionAgreement.objects.filter(published=True))
```